### PR TITLE
Bug 1874081: Fixed 4.4->4.5 upgrade test

### PR DESCRIPTION
**Pull request by igor-karpukhin**
Removed usage of associative arrays for upgrade test as they don't work properly for bash 4.2

### CODE REVIEW

File: hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh [hunk 1]
```diff
@@ -66,6 +66,13 @@ get_es_indices() {
     | xargs -I '{}' oc -n openshift-operators-redhat exec '{}' -c elasticsearch -- es_util --query=_cat/indices
 }
 
+get_es_indices_names() {
+  oc -n openshift-operators-redhat get pods -l component=elasticsearch --no-headers=true --ignore-not-found \
```
Comment (ewolinetz) [inline hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh:70]: we don't have ES pods deployed in this namespace at all. the operator is in `openshift-operators-redhat`

Comment (igor-karpukhin) [reply hack/testing-olm-upgrade/test-020-olm-upgrade-4.4-4.5.sh:70]: In general, we don't, but this test deploys EO into that namespace for simplicity.

**Pull request merged**